com.github.Bleuzen.FFaudioConverter.json

```diff
@@ -0,0 +1,79 @@
+{
+    "app-id": "com.github.Bleuzen.FFaudioConverter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "ffaudioconverter",
+    "rename-desktop-file": "ffaudioconverter.desktop",
+    "rename-appdata-file": "FFaudioConverter.appdata.xml",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--filesystem=host"
+    ],
+    "modules": [
+        {
+            "name": "libmp3lame",
+            "config-opts": ["--enable-shared",
+                            "--enable-nasm",
+                            "--bindir=/app/bin"],
+            "cleanup": [
+                "/share/doc",
+                "/share/man",
+                "/include"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/lame/lame-3.100.tar.gz",
+                    "sha256": "ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e"
+                }
+            ]
+        },
+        {
+            "name": "ffmpeg",
+            "config-opts": [
+                "--enable-gpl",
+                "--disable-static",
+                "--enable-shared",
+                "--disable-doc",
+                "--disable-ffplay",
+                "--disable-ffprobe",
+                "--enable-libmp3lame",
+                "--enable-libvorbis",
+                "--enable-libopus"
+            ],
+            "cleanup": [
+                "/share/ffmpeg/examples"
+            ],
+            "sources": [
+                {
+                "type": "archive",
+                "url": "https://ffmpeg.org/releases/ffmpeg-4.1.tar.xz",
+                "sha256": "a38ec4d026efb58506a99ad5cd23d5a9793b4bf415f2c4c2e9c1bb444acd1994"
+                }
+            ]
+        },
+        {
+            "name": "FFaudioConverter",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://github.com/Bleuzen/FFaudioConverter/releases/download/v0.10.0/FFaudioConverter-0.10.0.tar.xz",
+                    "sha256": "39590364a35c2203f37359a7d081e825ddbd034baf474250c80a2ca9900836a5"
+                }
+            ],
+            "build-commands": [
+                "tar xf FFaudioConverter-0.10.0.tar.xz",
+                "sed -i 's:/usr/bin/ffmpeg:/app/bin/ffmpeg:g' src/settings.cpp",
+                "./build.sh",
+                "install -D build/dist/ffaudioconverter /app/bin/ffaudioconverter",
+                "install -D ffaudioconverter.desktop /app/share/applications/ffaudioconverter.desktop",
```

Comment:
Your build system (`qmake`) should be installing these files.

Reply:
done